sycl/doc/design/spirv-extensions/SPV_INTEL_joint_matrix.asciidoc

```diff
@@ -145,12 +145,23 @@ Add section 3.XX, Matrix layout.
 2+^| Layout ^| Enabling capability 
 | 0 | *ColumnMajor*            |  *{capability_name}*
 | 1 | *RowMajor*               |  *{capability_name}*
-| 2 | *PackedA* +
-Suitable for VNNI instructions | *{capability_name}*
-| 3 | *PackedB* +
+| 2 | *Packed* +
 Suitable for VNNI instructions | *{capability_name}*
 |====
 
+=== Matrix Use
+
+Add section 3.XX, Matrix Use.
+
+[options="header"]
+|====
+2+^| Use ^| Enabling capability
+| 0 | *MatrixA*     | *{capability_name}*
+| 1 | *MatrixB*     | *{capability_name}*
+| 2 | *Accumulator* | *{capability_name}*
+| 3 | *Unnecessary* | *{capability_name}*
```

Comment:
The **Unnecessary** use is never documented anywhere in this extension.  Should it be?

Reply:
That's actually a spec bug, there shouldn't be any `Unnecessary` value for Use parameter, instead there should be `Unused` Layout value. I'll update the spec